main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"game/board"
+	"strings"
+)
+const size = 20
```

Comment:
The consts that starts with small letter are internal, and the ones that starts with capital letter are exported. Does the MaxRound needs to be exported?